bittensor/_subtensor/__init__.py

```diff
@@ -102,7 +105,7 @@ def __new__(
         config.subtensor._mock = _mock if _mock != None else config.subtensor._mock
         if config.subtensor._mock == True or network == 'mock' or config.subtensor.network == 'mock':
```

Comment:
all these `if __ OR __ OR __` still create room for erroneously returning a mock.. is there some way we can make it more stringent?